streamwebs_frontend/streamwebs/models.py

```diff
@@ -278,6 +278,30 @@ class Macroinvertebrates(models.Model):
     def __str__(self):
         return self.site.site_name
 
+    def clean(self):
+        if ((self.caddisfly + self.mayfly + self.riffle_beetle
+             + self.stonefly + self.water_penny +
+             self.dobsonfly) * 3) != self.sensitive_total:
+                raise ValidationError(
+                    _('%(sensitive_total)s is not the correct total'),
+                    params={'sensitive_total': self.sensitive_total},
+                )
+
+        if((self.clam_or_mussel + self.crane_fly + self.crayfish
+           + self.damselfly + self.dragonfly + self.scud + self.fishfly
+           + self.alderfly + self.mite) * 2) != self.somewhat_sensitive_total:
+                raise ValidationError(
+                    _('%(sensitive_total)s is not the correct total'),
+                    params={'sensitive_total': self.somewhat_sensitive_total},
```

Comment:
I think you want ``%(somewhat_sensitive_total)`` instead of ``%(sensitive_total)`` as the thing to print out in the error?